app/views/spree/admin/shipping_methods/_form.html.haml

```diff
@@ -53,7 +53,7 @@
       = f.field_container :categories do
         - Spree::ShippingCategory.all.each do |category|
           = label_tag do
-            = check_box_tag('shipping_method[shipping_categories][]', category.id, @shipping_method.shipping_categories.include?(category))
+            = check_box_tag('shipping_method[shipping_categories][]', category.id, category == DefaultShippingCategory.find_or_create)
```

Comment:
Same issue here - the "Default" checkbox is the only one selected when you load the edit shipping method page, even when it has been associated to all available shipping methods.

Reply:
Thanks again for pointing this out! Since the shipping method accepts multiple categories, I used this variation to instantiate the class:

`def new
        @object.shipping_categories = [DefaultShippingCategory.find_or_create]
        super
end`